### DISCUSSION TITLE
Building security onion iso file

### DISCUSSION THREAD

**Discussion by sebeheshtian** [Security-Onion-Solutions/securityonion#12671] [2.4]
### Version

2.4.60

### Installation Method

Other (please provide detail below)

### Description

other (please provide detail below)

### Installation Type

other (please provide detail below)

### Location

airgap

### Hardware Specs

other (please provide detail below)

### CPU

16

### RAM

40

### Storage for /

200

### Storage for /nsm

100

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi there,
I wanted to build an iso file from security onion sources and I want to know how can I do that?
thank you so much

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
The official Security Onion ISO is already available for download at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md

https://docs.securityonion.net/en/2.4/download.html#download

**Reply by sebeheshtian** [reply]
Thanks you for the answer. I want to make some changes and make the ISO from the start. If I remember correctly there was a build.sh in previous versions of security onion repository but it is missing from 2.4.60 version.
